src/server/req_runjob.c

```diff
@@ -545,6 +546,10 @@ req_runjob(struct batch_request *preq)
 		if (pjob) {
 			/* free prov_vnode before use */
 			job_attr_def[(int) JOB_ATR_prov_vnode].at_free(&pjob->ji_wattr[(int) JOB_ATR_prov_vnode]);
+			ptbl = parent->ji_ajtrk;
+			if (ptbl == NULL)
+				return;
```

Comment:
I don't think you should return/bail here, this is just a performance optimization, so just add the flag if parent->ji_ajtrk exists so that functionally it still works like before.